<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEditor</class>
 <widget class="QDialog" name="TagEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择你的个人特点</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 245, 245);
</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="ScrollContent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>239</width>
      <height>289</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>匹配好友</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
